checkout_sdk/risk/risk_client.py

```diff
@@ -15,6 +17,10 @@ def __init__(self, api_client: ApiClient, configuration: CheckoutConfiguration):
         super().__init__(api_client=api_client,
                          configuration=configuration,
                          authorization_type=AuthorizationType.SECRET_KEY)
+        warn(
+            'Risk endpoints have been officially no longer supported, This module will be removed in a future release.',
```

Comment:
I think that `Risk endpoints are no longer supported officially` would be better